test/assert.js

```diff
@@ -1922,6 +1946,20 @@ describe('assert', function () {
       if (typeof Symbol === 'function') {
         assert[isSealed](Symbol());
       }
+
+      if (typeof Proxy === 'function') {
+        var proxy = new Proxy({}, {
+          ownKeys: function() {
+            throw new TypeError();
+          }
+        });
+
+        Object.preventExtensions(proxy);
+
+        err(function() {
+          assert[isSealed](proxy);
+        }, { name: 'TypeError' });
+      }
```

Comment:
I think the above test is advanced enough that it calls for a short comment explaining why it is the way it is, particularly with regard to the `preventExtensions` and `ownKeys`. A comment will lower the barrier of entry to new contributors and reduce risk of someone submitting a change to it without realizing how it works internally.